mandatory/1-syntax-errors.js

```diff
@@ -1,16 +1,16 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
   return a + b + c;
 }
 
-function introduceMe(name, age)
-  return "Hello, my name is " + name "and I am " age + "years old";
-
+function introduceMe(name, age){
+  return "Hello, my name is " + name +" "+ "and I am "+ age +" "+ "years old";
```

Comment:
If you use the Prettier extension in VS Code and enable the editor format on save and editor format on paste settings, every time you save your code, Prettier will automatically format the code to look better so that it is easier to read. so this line would turn into: `return "Hello, my name is " + name + "and I am " + age + "years old";`

you also need to run the tests in the terminal by using `npm test -- --testPathPattern 1-syntax-errors` but make sure that you have node installed first and you have ran `npm install` at the root of the project folder so that Jest is installed which what is used to run those tests. The test won't pass till the string output is exactly as the test output.